### DISCUSSION TITLE
Make a column span based on a condition

### DISCUSSION THREAD

**Discussion by haztakki** [filamentphp/filament#3506] [Help] [answered]
Is it possible to make a column span based on a condition? For the first group, the select should fill span both columns as the text input is hidden.

<img width="540" alt="Screenshot 2022-08-15 at 15 06 40" src="https://user-images.githubusercontent.com/20573863/184632308-1c2badef-55e0-434c-a126-b4e5aaf311fa.png">

**Comment by danharrin** [comment, marked as answer]
You can pass a closure to the columnSpan() and use $get to switch between 1 and 2 columns?

**Reply by haztakki** [reply]
Thanks @danharrin! Works a treat. Much appreciated.
